poolqc/pooling_qc_sample.py

```diff
@@ -0,0 +1,291 @@
+import gsiqcetl.bcl2fastq.parse
+import gsiqcetl.bcl2fastq.utility
+import dash_core_components as dcc
+import dash_table.FormatTemplate as FormatTemplate
+import dash_html_components as html
+import dash.dependencies as dep
+import dash_table as dt
+import pandas
+import plotly.plotly as plotly
+import plotly.graph_objs as go
+import numpy as np
+import urllib
+
+pandas.options.mode.chained_assignment = None
+
+rnaseq = pandas.read_hdf('./data/rnaseqqc_cache.hd5')
+rnaseq.rename(columns={'Sample Name': 'library'}, inplace=True)
+
+bamqc = pandas.read_hdf('./data/bamqc_cache.hd5')
+
+
+bcl2fastq = gsiqcetl.bcl2fastq.parse.load_cache(
+    gsiqcetl.bcl2fastq.parse.CACHENAME.SAMPLES,
+    './data/bcl2fastq_cache.hd5')
+bcl2fastq['library'] = bcl2fastq['SampleID'].str.extract('SWID_\d+_(\w+_\d+_.*_\d+_[A-Z]{2})_')
+
+df = bcl2fastq.merge(rnaseq, on='library', how='outer')
+df = df.merge(bamqc, on='library', how='outer')
+
+df.drop(columns=['Sequencer Run Name', 'Lane Number'])
```

Comment:
You can also set the `inplace` parameter to `True`, but I prefer re-assignment, as it is visually more obvious what you are doing.

Reply:
I think it might be beneficial to keep these columns and drop them in specific graph functions. This will make it easier to differentiate between RNA and DNA graphs as we can then filter based on whether these columns are empty or not. ex. line 221 in next commit

Reply:
Why choose these columns to differentiate between RNA and DNA, as opposed to columns like "rRNA Contamination"?

Reply:
Since there are times when the Run Name for rnaseq is blank, this reduces the need to filter those out later.